packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.ts

```diff
@@ -70,6 +70,13 @@ export abstract class XYChartComponent extends TimeseriesChartComponent
         return this.configuration.legendPlacement === LegendPlacement.Right;
     }
 
+    /** Checks if the series is interactive or if the dataSource has a link property */
+    public isSeriesInteractive(legendSeries: IChartAssistSeries<IAccessors>): boolean {
+        const link: boolean = !(legendSeries?.link === undefined) || !(this.seriesInteractive === undefined);
+        return link;
```

Comment:
could be simplified to the following:

```suggestion
        return !(legendSeries?.link === undefined) || !(this.seriesInteractive === undefined);
```